### ISSUE TITLE
[BO] Bad display of the dropdown menu icons in header search for legacy theme

### ISSUE THREAD

**Issue by mparvazi** [PrestaShop/PrestaShop#26624]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In BO pages that designed by legacy theme (e.g. Dashboard), if user select a search type in header search dropdown menu, selected icon will be displayed incorrectly.

**Screenshot:**

<img src="https://user-images.githubusercontent.com/85633460/141962188-a80ccf21-d99c-4b23-adc2-2e63be2b0237.png" width="400"/>

[https://watch.screencastify.com/v/hSIMZfY8u4BVmr4hW0ha](https://watch.screencastify.com/v/hSIMZfY8u4BVmr4hW0ha)

### Expected behavior

When a search type is selected from dropdown menu, only its icon must be displayed in the header search box.

### Steps to reproduce

1. Go to BO > Dashboard
2. Click the search icon in header search box
3. Select a search type from the menu

### PrestaShop version(s) where the bug happened

1.7.8.1

### PHP version(s) where the bug happened

7.4

**Comment by Hlavtox** [comment]
@NeOMakinG ~~wrong icon name~~ It changes the class from material-icons to icon-book and the "search" icon name is kept in the <i> tag.

**Comment by mparvazi** [comment]
There is a mismatch between jquery script and template.
The jquery script use the font-awesome icon but template use the material icons.

**Comment by mparvazi** [comment]
**\themes\default\template\search_form.tpl:**

The icon elements `<i></i>` must be defined by material icons.
The data-icon must have the name of material icon.
```
<ul id="header_search_options" class="dropdown-menu">
  <li class="search-orders search-option">
    <a href="#" data-value="3" data-placeholder="{l s='Order ID' d='Admin.Navigation.Header'}" data-icon="shopping_basket">
      <i class="material-icons">shopping_basket</i> {l s='Orders' d='Admin.Global'}
    </a>
  </li>
</ul>
```

**\themes\default\js\admin-theme.js**

`this` = An option that selected from dropdown menu.
```
$('#search_type_icon').text($(this).data('icon'));
```
**\themes\default\scss\partials\\_header.scss**

This code unifies the appearance of dropdown menu between legacy and new theme.
```
.dropdown-menu {
  .material-icons {
    font-size: 18px;
    line-height: 18px;
    padding-right: 0.5rem;
    margin-right: 0.75rem;
    color: #6c868e;
    vertical-align: text-bottom !important;
  }

  .active {
    .material-icons {
      color: #fff;
    }
  }
}
```

[https://watch.screencastify.com/v/cBDgjrzHMJuYRIPUV9lc](https://watch.screencastify.com/v/cBDgjrzHMJuYRIPUV9lc)

**Comment by hibatallahAouadni** [comment]
Hello @mparvazi 

I was able to reproduce the issue with PrestaShop version 1.7.6.7, 1.7.7.8, 1.7.8.0 and develop. I’ll add this to the debug backlog so that it’s fixed.

Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you. 

If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

**Comment by hibatallahAouadni** [comment]
@Hlavtox @mparvazi I think this issue will be fixed when we migrate the legacy pages :wink: 
Ping @PrestaShop/product-team & @NeOMakinG shall we fix the issue on legacy pages or it's not necessary?

**Comment by NeOMakinG** [comment]
> Hello @mparvazi
> 
> I was able to reproduce the issue with PrestaShop version 1.7.6.7, 1.7.7.8, 1.7.8.0 and develop. I’ll add this to the debug backlog so that it’s fixed.
> 
> Please be aware that due to the high number of reported issues, some bugs might take a very long time to be resolved. If this issue is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you.
> 
> If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.
> 
> Thank you

We've no idea of the date of the end of the dashboard migration, I think we should consider fixing it

But maybe we should let the @PrestaShop/product-team decide

**Comment by mparvazi** [comment]
It'll be fixed as a part of bundle for minor front-end problems in BO.

**Comment by mparvazi** [comment]
@marionf 

I think this issue should be closed, because probably it will be fixed by #27185

**Comment by hibatallahAouadni** [comment]
@mparvazi why you closed the issue? did you fix it? if so, where's the PR?

**Comment by mparvazi** [comment]
Hello @hibatallahAouadni 
It will be fixed here: #27185

**Comment by hibatallahAouadni** [comment]
Thanks for the tip :wink:

**Comment by hibatallahAouadni** [comment]
Duplicate of #27185

**Comment by mparvazi** [comment]
No one trust me here about what I do 😭

**Comment by hibatallahAouadni** [comment]
@mparvazi That's not true, we trust you :muscle: and we're waiting for your PR to fix this issue #27185 :superhero: 
It's just we create new issue to add more details to be easy for you or any other contributor to fix the problem.
